### DISCUSSION TITLE
Plotting CartPole environment in Jupyter

### DISCUSSION THREAD

**Discussion by jamblejoe** [JuliaReinforcementLearning/ReinforcementLearning.jl#307] [Q&A]
```
cartpole_env = ReinforcementLearningEnvironments.CartPoleEnv()
plot(cartpole_env)
plot()
```
Last `plot()` is needed to invoke plotting. `GR.plot(env::CartPoleEnv)` does return `nothing`, so no plot is shown in the notebook, I would fix but I dont know how `GR` works.

**Comment by findmyway** [comment]
You can simply call `RLEnvs.plot(cartpole_env)`.

I'll work on https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/issues/246 soon.
